### PR TITLE
Img_processing: Add optional travel status

### PR DESCRIPTION
**Pull request by TziporaGoldshmidt**

### CODE REVIEW

File: img_processing/src/manager.cpp [hunk 1]
```diff
@@ -41,40 +47,48 @@ void Manager::init()
 
 void Manager::mainDemo()
 {
-    string filePath = "../data.txt";
-    // open the file
-    ifstream file(filePath);
+    string path;
+    int focalLength;
+    bool isTravel;
+    // Open the JSON file
```
Comment (ChedvaErblich) [inline img_processing/src/manager.cpp:51]: same line twice

File: img_processing/src/manager.cpp [hunk 2]
```diff
@@ -41,40 +47,48 @@ void Manager::init()
...
+    // Open the JSON file
+    std::ifstream file("../data.json");
+
     if (!file.is_open()) {
-        LogManager::logErrorMessage(ErrorType::FILE_ERROR);
+        std::cerr << "Failed to open the file." << std::endl;
```
Comment (ChedvaErblich) [inline img_processing/src/manager.cpp]: Why/?
Use the logger

File: img_processing/data.json
Comment (ChedvaErblich) [inline img_processing/data.json:1]: This file should be given via run time, your code should support any video,
if you need this for testing, put this file under the test dir

File: img_processing/src/manager.cpp [hunk 3]
```diff
@@ -3,6 +3,12 @@
 #include "manager.h"
 #include "alert.h"
```
Comment (ChedvaErblich) [inline img_processing/src/manager.cpp]: Includes are not in the right order

**Review by ChedvaErblich — CHANGES_REQUESTED** [review]
Please have someone else review the code, and after she approves it, I'll push it in.

File: img_processing/src/manager.cpp (hunk 1 above)
Comment (TziporaGoldshmidt) [reply img_processing/src/manager.cpp:51]: done

File: img_processing/src/manager.cpp (hunk 2 above)
Comment (TziporaGoldshmidt) [reply img_processing/src/manager.cpp]: done

File: img_processing/src/manager.cpp (hunk 3 above)
Comment (TziporaGoldshmidt) [reply img_processing/src/manager.cpp]: done

**Pull request closed without merging**